ci-operator/step-registry/azure/provision/role-assignment/hypershift/azure-provision-role-assignment-hypershift-ref.yaml

```diff
@@ -0,0 +1,22 @@
+ref:
+  as: azure-provision-role-assignment-hypershift
+  from_image:
+    namespace: ocp
+    name: "4.16"
```

Comment:
Should this be updated?

Reply:
It makes no difference really since were only using it for the az cli. The most recent version I see being used is 4.17 so we can bump to that.